docs/src/systems/implicit_incompressible_sph.md

```diff
@@ -0,0 +1,233 @@
+# [Implicit Incompressible SPH](@id iisph)
+
+Implicit Incompressible SPH as introduced by [M. Ihmsen](@cite ihmsen2013) is a scheme that computes pressure values, by iteratively solving a linear system using a relaxed jacobi system, to resolve the particles density deviation from the reference density. This method solves a linear system $Ax=b$ where the pressure values that are used for the pressure acceleration are the unknown variable $x$.
+It does not use a state equation to generate pressure values like the [weakly compressible SPH scheme](weakly_compressible_sph.md).
+
+
+To derive the formulation for the linear system we start by discretizing the Continuity equation
+
+```math
+\frac{D\rho}{Dt} = - \rho \nabla \cdot v
+```
+
+For a particle $i$ a forward difference method is used to discretizie the left hand side of the equation to get
+
+```math
+\frac{\rho_i(t+ \Delta t - \rho_i(t))}{\Delta t}.
+```
+
+The right-hand side is discretized by the difference formulation of the divergence of the velocity field.
+
+```math
+\nabla \cdot \textbf{v}_i = \frac{1}{\rho_i} \sum_j m_j \textbf{v}_{ij} \nabla W_{ij},
+```
+where $\textbf{v}_{ij}$ is equal to $\textbf{v}_i - \textbf{v}_j$.
+
+So all in all the following discretized version of the continuity equation for a particle $i$ is achieved:
+
+```math
+\frac{\rho_i(t + \Delta t) - \rho_i(t)}{\Delta t} = \sum_j m_j \textbf{v}_{ij}(t+\Delta t) \nabla W_{ij}
+```
+
+In this implicit formulation the unknown velocities $\textbf{v}_{ij}(t + \Delta t)$ are needed.
+
+They are given by adding all pressure and non-pressure accelerations to the current velocity:
+
+```math
+\textbf{v}_i(t + \Delta t) = \textbf{v}_i(t) * \frac{\textbf{F}_i^{adv}(t) + \textbf{F}_i^p(t)}{m_i}
+```
```

Comment:
````suggestion
Using the semi-implicit Euler method, we can obtain the velocity in the next time step as
```math
\bm{v}_i(t + \Delta t) = \bm{v}_i(t) + \Deltat t \frac{\bm{F}_i^\text{adv}(t) + \bm{F}_i^p(t)}{m_i}
```
````
Note the `^\text{adv}`. We are in math mode, so `adv` means a times d times v. This is then rendered as italic and with different kerning than text.
Since you mean the text "adv", this should be put inside a `\text`. This mistake is very common and is often found in papers as well, such as the Ihmsen 2013 paper.